. prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
Sorry for not mentioning any rules we wanted but let's go with singleQuote: false, which I think is the prettier default.